portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,86 @@
+<!-- 
+MAIN HTML
```

Comment:
This info seems like reasonable stuff to keep in the repo for your own reference, but maybe consider moving it to another file, something like a README or notes file.